### ISSUE TITLE
Flicker on static or barely moved sources between complex scenes

### ISSUE THREAD

**Issue by NOVAinc** [exeldro/obs-move-transition#99]
Hey there! I've been using move transition and I love the potential! There are a few issues I ran into, the other one I reported already and then there's this one.

When the same source is present on two complex scenes with the same or near same position, they flicker when transitioning from one scene to the other. Here is a short clip to demonstrate the issue.

https://user-images.githubusercontent.com/19545604/141597843-81863843-55cc-4a59-ba36-acc10fb8b474.mp4

As you can see, the dark orange bar behind my logo (and many other static or barely moving sources) flickers noticeably when moving from one scene to the other. Here are the general settings for move transition.

![image](https://user-images.githubusercontent.com/19545604/141597973-bf501972-e856-4457-b4cc-d98fa8c25f32.png)

I have a move transition override filter for that source, but it has default values for everything within "matched source" section. I only have overrides for when appearing/disappearing.

If you need any additional information please let me know.

**Comment by exeldro** [comment]
What kind of sources are the ones with the problem, image source?

**Comment by NOVAinc** [comment]
Correct sir.

**Comment by exeldro** [comment]
I can't seem to replicate it, can you provide me your scene collection to try and test it?

**Comment by NOVAinc** [comment]
I downloaded the portable beta version of OBS as well as the most up-to-date move transition plug-in, and I've been unable to replicate the issue myself on a first attempt. I'll install the streamFX plug-in on the beta to try and use source mirrors (could be a bug on their end?) and I'll report back!

**Comment by NOVAinc** [comment]
> I can't seem to replicate it, can you provide me your scene collection to try and test it?

@exeldro I have tested it again with the latest release on the latest obs beta and I found it was present on this configuration. Let me know if you see the same issue. I'm using the portable version of OBS with only the move transition plugin, no streamFX 

[bug report.zip](https://github.com/exeldro/obs-move-transition/files/7831702/bug.report.zip)

Here is a video of what it looks like on my end.

https://user-images.githubusercontent.com/19545604/148617155-9f119ffd-56fe-4f5f-a371-641d3f37bf91.mp4

Thanks for making and maintaining this plug-in! It really helps provide good production value in a straightforward and accessible way. I have starred your project.
